.travis.yml

```diff
@@ -16,4 +16,6 @@ install:
 - bundle install
 
 script:
-- bundle exec jekyll build
+- bundle exec jekyll build 2> error.log
+- cat >&2 error.log
+- grep -qie Error error.log && false || true
```

Comment:
How should I read this line? Just from reading it it seems like:
```
true && false || true ==
(true && false) || true ==
false || true ==
true
```
and
``` 
false && false || true ==
(false && false) || true ==
false || true ==
true
```
So either way it will return true.
I fully admit I maybe missing something here.